### ISSUE TITLE
"Docker Engine stopped" on Intel iMac running Ventura 13.5, Docker version 4.22.0

### ISSUE THREAD

**Issue by sundarjhu** [docker/for-mac#6941]
### Description

Docked Desktop UI opens, able to access settings and edit them, but that's it. I get the "Docker Engine stopped" message and nothing else. Command line checks result in "ERROR: Cannot connect to the Docker daemon. Is the docker daemon running?" messages.

### Reproduce

Intel iMac updated to Ventura 13.5
Install and run Docker Desktop

### Expected behavior

Docker Engine should start up when Docker Desktop is opened

### docker version

```bash
Cannot connect to the Docker daemon at unix:///Volumes/.docker/run/docker.sock. Is the docker daemon running?
Client:
 Cloud integration: v1.0.35-desktop+001
 Version:           24.0.5
 API version:       1.43
 Go version:        go1.20.6
 Git commit:        ced0996
 Built:             Fri Jul 21 20:32:30 2023
 OS/Arch:           darwin/amd64
 Context:           desktop-linux
```

### docker info

```bash
Client:
 Version:    24.0.5
 Context:    desktop-linux
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc.)
    Version:  v0.11.2-desktop.1
    Path:     /Volumes/.docker/cli-plugins/docker-buildx
  compose: Docker Compose (Docker Inc.)
    Version:  v2.20.2-desktop.1
    Path:     /Volumes/.docker/cli-plugins/docker-compose
  extension: Manages Docker extensions (Docker Inc.)
    Version:  v0.2.20
    Path:     /Volumes/.docker/cli-plugins/docker-extension
  sbom: View the packaged-based Software Bill Of Materials (SBOM) for an image (Anchore Inc.)
    Version:  0.6.0
    Path:     /Volumes/.docker/cli-plugins/docker-sbom
  scan: Docker Scan (Docker Inc.)
    Version:  v0.26.0
    Path:     /Volumes/.docker/cli-plugins/docker-scan
  scout: Command line tool for Docker Scout (Docker Inc.)
    Version:  0.20.0
    Path:     /Volumes/.docker/cli-plugins/docker-scout
WARNING: Plugin "/Volumes/.docker/cli-plugins/docker-dev" is not valid: failed to fetch metadata: exit status 2
WARNING: Plugin "/Volumes/.docker/cli-plugins/docker-init" is not valid: failed to fetch metadata: exit status 2

Server:
ERROR: Cannot connect to the Docker daemon at unix:///Volumes/.docker/run/docker.sock. Is the docker daemon running?
errors pretty printing info
```

### Diagnostics ID

FC9CA353-CFD2-42EB-8177-3CBBF0FA4644/20230812005745

**Comment by waded** [comment]
Mac M1 here, Venture 13.5.1, same symptoms and same docker version / docker info (with the exception of darwin/arm64 reported for OS/arch, and the dev & init plugins are not showing a warning in my case.) A vote for possibly not architecture specific if that matters for triage.

Diagnostics ID: E475ECB1-F272-40E3-8011-613FB27FF575/20230819233001

**Comment by ctalledo** [comment]
Hi folks, 

We've addressed that problem in a [new v4.22.1 patch release](https://docs.docker.com/desktop/release-notes/).

Could you please give that a try and let us know if the issue is fixed or not?

Thanks!

**Comment by ctalledo** [comment]
Closing as duplicate of #6936.
